README.md

```diff
@@ -1 +1,62 @@
-# Awesomebooks
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Awesome Books: plain JavaScript with objects
+
+> In this project, we will build a basic website that allows users to add/remove books from a list. We will achieve that by using JavaScript objects and arrays. We will also need to dynamically modify the DOM and add basic events.
+
+
+
+### Links
+
+- Live Site URL: [see live demo](https://cvalencia1991.github.io/Awesomebooks/)
+
+
+
+## Built With
+
+- Languages used HTML, CSS, Javascript.
+- Technologies used Github, Git, VSCode.
+
+
+
```

Comment:
- [ ] I think it would be a good idea if you add the `Getting start` part to your `README` file and explain clearly how users can clone and open your project locally, also highlights what users need in their machine to start a project so that if some need to clone your repo and they don't know how to do it they can just follow the instructions step by step in your `README` file 😉 😉

Reply:
ok you are right, i add it in the README.md File